src/input/hotkey.c

```diff
@@ -150,85 +150,94 @@ void hotkey_character(int c)
 {
     if (data.alt_down) {
         switch (c) {
-            case 'X': case 'x':
+            case SDLK_x: //'X'
```

Comment:
Please, *no* SDL-related code/constants in anything other than the `platform` directory.